### PR TITLE
CLC-5341 Shell script to report LTI app visibility

### PR DESCRIPTION
**Pull request by raydavis**
https://jira.ets.berkeley.edu/jira/browse/CLC-5341
Missed this little file in the previous PR...

### CODE REVIEW

File: script/report-lti-usage.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+cd $( dirname "${BASH_SOURCE[0]}" )/..
+
+LOG=`date +"$PWD/log/report_lti_usage_%Y-%m-%d.log"`
+LOGIT="tee -a $LOG"
+
+# Enable rvm and use the correct Ruby version and gem set.
+[[ -s "$HOME/.rvm/scripts/rvm" ]] && . "$HOME/.rvm/scripts/rvm"
+source .rvmrc
+
+export RAILS_ENV=production
```
Comment (johncrossman) [inline script/report-lti-usage.sh]: with the OEC-related shell script I used the following. Should we allow devs to override RAILS_ENV (e.g., 'testext')?
``export RAILS_ENV=${RAILS_ENV:-production}``

File: script/report-lti-usage.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+export LOGGER_LEVEL=INFO
+export JRUBY_OPTS="-Xcext.enabled=true -J-client -X-C"
+
+echo | $LOGIT
+echo "------------------------------------------" | $LOGIT
+echo "`date`: About to run the LTI usage reporting script..." | $LOGIT
+
+cd deploy
+
+bundle exec rake canvas:report_lti_usage | $LOGIT
```
Comment (johncrossman) [inline script/report-lti-usage.sh]: Another nit-pick (and prolly doesn't matter): IMO, a line break at end-of-file is a good standard.

File: script/report-lti-usage.sh (hunk 1 above)
Comment (raydavis) [inline script/report-lti-usage.sh]: Hmm, good question. Our existing pattern sets "production" directly in the shell scripts, but it's true that our shared servers don't have a RAILS_ENV by default. How about if we leave this as is and do a follow-up to cover all the scripts?

Comment (johncrossman) [inline script/report-lti-usage.sh]: Fine by me, @raydavis

Comment (raydavis) [inline script/report-lti-usage.sh]: Added CLC-5842 for the RAILS_ENV enabler.

**Comment by johncrossman** [comment]
Looks good, @raydavis

**Pull request merged**